pc/rpc_client.hpp

```diff
@@ -450,22 +452,16 @@ namespace pc
       template<class T>
       size_t get_data_val( T *, size_t srclen=sizeof(T) ) const;
 
-      account_subscribe();
-      void request( json_wtr& ) override;
-      void response( const jtree& ) override;
-      bool notify( const jtree& ) override;
-
-    private:
-      pub_key    *acc_;
+    protected:
+      pub_key     acc_;
```

Comment:
Why did this change from a `pub_key*` to a `pub_key`?

Reply:
because it's an output of the program calls whereas in the account_subscribe it's a parameter